pkg/volume/csi/csi_test.go

```diff
@@ -306,6 +307,7 @@ func TestCSI_VolumeAll(t *testing.T) {
 				// creates VolumeAttachment and blocks until it is marked attached (done by external attacher)
 				go func(spec *volume.Spec, nodeName types.NodeName) {
 					attachID, err := volAttacher.Attach(spec, nodeName)
+					waitForAttachDone <- attachID
```

Comment:
close(waitForAttachDone)  may be more good， my opinion. :)

Reply:
Thanks! Sorry I forget to do that :)

Reply:
It seems that you don't understand me. I still advise to use the close here. Because it's no blocking. This routine doesn't need to wait the others to receive it. On the other side, can you promise that the code in 568 line can execute?  it has a if judge before in line 555.if the code doesn't execute, this routine will block forever. So when we close it here, you don't worry about block scene and also you can  promise that Detach execute after the Attach. :)

Reply:
Yes, you are right. The routine blocked indeed.
I hope I can be more careful next time:(  .Thanks, again.

Reply:
it can be removed. I don't explain it clearly. sorry!  For the most start,  I advise that this line can be changed to `close(waitForAttachDone)`.